config/bundles/bluima.regions_0.1.json

```diff
@@ -0,0 +1,91 @@
+{
+  "name": "bluima_regions",
+  "domain": "bluima",
+  "version": "0.1",
+  "dependencies": [
+    {
+      "value": "ch.epfl.bbp.nlp:bluima_regions:0.1"
+    },
+    {
```

Comment:
I think utils and units are already a dependency of regions, so they should get pulled automatically, no?